Makefile

```diff
@@ -79,7 +79,7 @@ IMMUTABLE_DOCKER_TAG := $(VERSION)
 
 .PHONY: dep
```

Comment:
nit: I wonder if we should update the name of this target from `dep` to... `bootstrap`?

Reply:
When I originally created that make target, I saw it as coincidental that its name matches the dependency management tool we were using. It's job was (and still is) to manage the dependencies and I had felt `dep` was a succinct and descriptive name. imho, it still is.

fwiw, I am opposed to `bootstrap` because I feel it carries a connotation that it's a target that must be executed before one undertakes any development upon a repo (and perhaps needs to re-execute when switching branches?) Any need for such a thing has been obviated by the fact that we vendor our Go dependencies. So I've long been steering clear of the name `bootstrap` in an attempt to head off any possible confusion.

Reply:
Upon further reflection, I would be in favor of changing the name of this target to `resolve-dependencies`. It may be more verbose, but its function is crystal clear and does not (even coincidentally) hint at what the underlying dependency manager is.

Reply:
Good points all.  No strong opinion here; I'll leave it up to you.  I'm fine with leaving it as is, as well.